packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/execPermissionsWalkthrough.ts

```diff
@@ -274,7 +274,6 @@ const selectCategories = (choices: DistinctChoice<any>[], currentPermissionMap:
   name: 'categories',
   message: 'Select the categories you want this function to have access to.',
   choices,
-  validate: answers => (_.isEmpty(answers) ? 'You must select at least one category' : true),
```

Comment:
Can you please verify the flow where you add function -> Select add permission -> Don't select any category -> push? I think that flow would break with this change.

Reply:
Did we verify this flow?

Reply:
I verified the flow. It seems to be working perfectly.